README.md

```diff
@@ -171,7 +171,7 @@ that your company deserves a spot here, reach out to
             <td>DataDetect</td>
             <td align="center"><img height="32px" src="https://raw.githubusercontent.com/ory/meta/master/static/adopters/datadetect.svg" alt="Datadetect"></td>
             <td><a href="https://unifiedglobalarchiving.com/data-detect/">unifiedglobalarchiving.com/data-detect/</a></td>
-        </tr>        
```

Comment:
This needs to be done here: https://github.com/ory/meta/blob/master/templates/repository/common/PROJECTS.md :)

Reply:
Reverted this file.